### PR TITLE
fix: throw error when an egg plugin test is using bootstrap

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

close https://github.com/eggjs/egg/issues/3677

~~单测不知道怎么写才优雅，bootstrap 这个会被 cache 掉。~~

### CODE REVIEW

**Comment by fengmk2** [comment]
@atian25 单测运行完在 after hook 里面将 require cache 删除了。

File: bootstrap.js [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
 
 const assert = require('power-assert');
+const path = require('path');
 const mock = require('./index').default;
 
 const options = {};
 if (process.env.EGG_BASE_DIR) options.baseDir = process.env.EGG_BASE_DIR;
+
+// throw error when an egg plugin test is using bootstrap
+const pkgInfo = require(path.join(options.baseDir || process.cwd(), 'package.json'));
```
Comment (atian25) [inline bootstrap.js]: 不写单测果然有问题，刚才写成了 `require(path.join(cwd), 'package.json')`

**Comment by atian25** [comment]
[捂脸] 看来不是 appveyor 的问题，travis windows 也一样有权限问题，cc @popomore 

```bash
2019-05-06 03:10:18,134 ERROR 2884 [egg-logger] [C:\Users\travis\build\eggjs\egg-mock\test\fixtures\demo\logs\demo\common-error.log] Error: EPERM: operation not permitted, open 'C:\Users\travis\build\eggjs\egg-mock\test\fixtures\demo\logs\demo\common-error.log'
```

**Comment by atian25** [comment]
3.22.3

**Pull request merged**